.github/workflows/integration-tests.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Run tests
         run: |
           pip install pytest
-          python3 -m pip install -e sdk/python; pytest -s sdk/python/test --log-cli-level=debug --namespace=default
+          python3 -m pip install -e sdk/python; pytest -s sdk/python/test/e2e --log-cli-level=debug --namespace=default
```

Comment:
Since we create separate CPU-based image, can we use the `integration-test` GitHub action to run that test ?
You can add additional steps to this action to build Trainer + Storage Initializer.

Reply:
@andreyvelich Do you mean adding the e2e test for the train API directly in the `integration-tests.yaml` file, instead of running it as a separate GitHub Action?

Reply:
Yeah, I hope we have sufficient resources to run it.
Any thoughts @deepanker13 @saileshd1402 @kubeflow/wg-training-leads ?

Reply:
Since adding this test to `integration-test` would cause it to fail (as discussed below), I’ve moved it to a separate GitHub Action.